terraform/only_private_S3_buckets/only_private_S3_buckets.rego

```diff
@@ -9,4 +9,4 @@ deny[reason] {
     resources.type == "aws_s3_bucket"
     resources.change.after.acl != "private"
     reason:= "Deployment of not private S3 bucket is not allowed"
-}
+}
```

Comment:
No newline at the end of file. Don't think it should cause a problem, but please check this policy is working (via test is enough, I guess).

Reply:
Did you check this?